contracts/modules/cover/Cover.sol

```diff
@@ -21,6 +21,8 @@ import "../../libraries/SafeUintCast.sol";
 import "../../libraries/StakingPoolLibrary.sol";
 import "../../interfaces/IStakingProducts.sol";
 import "../../interfaces/ICoverProducts.sol";
+import "../../libraries/Math.sol";
+import "hardhat/console.sol";
```

Comment:
these imports and console.log statements will disappear from the final commit history by ammending the commit.

for convenience i benefit from having the logs here to debug certain parts like pricing

if it's an inconvenience when viewing the draft please let me know and i'll adapt